### DISCUSSION TITLE
Quiet mode?

### DISCUSSION THREAD

**Discussion by LeiChat** [signalkraft/mypyllant-component#39] [Q&A]
Hi. Is it possible to toggle a Vaillant ASHP's quiet mode through the integration/API?
(Or is that something you can schedule using the Vaillant controller/app, along with setback scheduling?)

**Comment by signalkraft** [comment]
I've only seen quiet mode settings in the advanced settings of the controller, I don't think these types of settings are available in the app (and therefore over the API).

**Reply by LeiChat** [reply]
Thank you.

Have seen a suggestion that enabling quiet mode is a way of throttling the heat pump, so it uses less electricity.
I was optimistically hoping that it might be something I can toggle through HA.

**Comment by rmalbrecht** [comment]
The quiet mode put's a limit on the upper modulation level. My Arotherm Plus 55 can modulate between 30% and 100%. If you enable the quiet mode, the upper modulation is limited to 60% or so. The concrete numbers are somewhere in the documentation and depend on the model. So, yes it might reduce energy consumption. But it also limits the heating capacity.
If your system is running at 50% or so anyway, because it's not really cold outside, then nothing changes.

**Comment by BenE-O** [comment]
Hi guys,
I'm definitely interested in quiet mode, in my case the AroTherm Plus
I've got a number of solar panels and battery storage.
During the night and "dark" hours I'd like to use the battery in quiet mode and when the sun shines the system can switch to full power.
Of course with the option of overruling the quiet mode should be an option.

I know this option is not in the API, but I asked for adoption of this setting. May be when a number of users are requesting this they might be willing to add this like the heating curve, which has been added not long ago.

I hope there are readers out here also interested

Thanks.
